### PR TITLE
feat: Add scc:privileged for aicoe-ci

### PR DESCRIPTION
**Pull request by tumido**
Resolves: https://github.com/operate-first/support/issues/81

### CODE REVIEW

**Review by 4n4nd — COMMENTED** [review]
/lgtm

**Review by harshad16 — COMMENTED** [review]
Thanks for the quick addition.
/lgtm

File: cluster-scope/base/clusterrolebindings/scc-privileged-opf-ci-pipelines-aicoe-ci/clusterrolebinding.yaml [hunk 1]
```diff
@@ -0,0 +1,13 @@
...
+metadata:
+  name: scc-privileged-opf-ci-pipelines-aicoe-ci
+roleRef:
+  apiGroup: rbac.authorization.k8s.io
+  kind: ClusterRole
+  name: system:openshift:scc:privileged
+subjects:
+  - kind: ServiceAccount
+    name: aicoe-ci
+    namespace: opf-ci-pipeline
```
Comment (HumairAK) [inline cluster-scope/base/clusterrolebindings/scc-privileged-opf-ci-pipelines-aicoe-ci/clusterrolebinding.yaml:13]: did you mean `opf-ci-pipelines` (the `s` at the end) ?

Comment (harshad16) [reply cluster-scope/base/clusterrolebindings/scc-privileged-opf-ci-pipelines-aicoe-ci/clusterrolebinding.yaml:13]: good catch, i missed that :0

Comment (tumido) [reply cluster-scope/base/clusterrolebindings/scc-privileged-opf-ci-pipelines-aicoe-ci/clusterrolebinding.yaml]: Hmm.. I think I meant precisely that. :smile: Thanks! :+1:

**Pull request merged**